src/scss/misc/_colors.scss

```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```

Comment:
This is a short term solution to allow recycling to be much clearer on the frontend. Our next set of changes to the icons/map will be much more comprehensive with different shapes and colors.